docs/resources/vault_cluster_admin_token.md

```diff
@@ -0,0 +1,56 @@
+---
+page_title: "hcp_vault_cluster_admin_token Resource - terraform-provider-hcp"
+subcategory: ""
+description: |-
+  The Vault cluster admin token resource generates an admin-level token for the HCP Vault cluster.
+---
+
+# Resource `hcp_vault_cluster_admin_token`
+
+~> **Important Security Notice** The admin token generated by this resource will
```

Comment:
Is there a similar problem with Consul tokens @bcmdarroch  ? 🤔 
I thought it was encrypted.

Reply:
It is but not in all cases: https://www.terraform.io/docs/language/state/sensitive-data.html